### ISSUE TITLE
Deployments off `prod` branch

### ISSUE THREAD

**Issue by jamilbk** [CloudFire-LLC/cloudfire#6241]
- Add `prod` branch
- Release clients off `prod`

**Comment by thomaseizinger** [comment]
Yey! I like git flow (treats the `master` branch as prod and merges into `dev`).

I once designed an entire release workflow with GitHub actions around that: https://github.com/thomaseizinger/github-action-gitflow-release-workflow

**Comment by jamilbk** [comment]
Yeah this was discussed Friday standup to help prevent the `ENABLE_MASQUERADE` thing from happening again. It's a large refactor of our CI pipeline, so instead we enabled Deployment approvers which you can click to see a diff.

**Comment by jamilbk** [comment]
This has been made irrelevant by doing deploys from a separate repo
